google/cloud/sql/connector/refresh_utils.py

```diff
@@ -190,11 +190,14 @@ async def _get_ephemeral(
 
     # decode cert to read expiration
     x509 = load_pem_x509_certificate(ephemeral_cert.encode("UTF-8"), default_backend())
-    expiration = x509.not_valid_after
+    expiration = x509.not_valid_after_utc
     # for IAM authentication OAuth2 token is embedded in cert so it
     # must still be valid for successful connection
     if enable_iam_auth:
         token_expiration: datetime.datetime = login_creds.expiry
+        # google.auth library strips timezone info for backwards compatibality
```

Comment:
And so we're going to re-add the timezone (which is UTC)?

When the auth library strips the timezone, do they assume UTC afterwords?

Also nit: this GitHub reference will become stale as soon as those lines change. Shall we link to a SHA?

Reply:
Yup we are going to re-add the timezone information to the datetime, its already in UTC based on the linked code. So adding it back should have no affect but I'll do a bit more research to be 100% sure that stripping and re-adding is okay.

Reply:
Sounds good. Let's just make the rationale clear in the code, so future maintainers don't trip over this.